civitai-downloader-v2/src/cli/main.py

```
@@ -169,8 +169,552 @@ async def run_search():
                 click.echo(f"\nResults saved to: {output}")
         
         except Exception as e:
-            click.echo(f"Search failed: {e}", err=True)
-            raise
+            click.echo(f"Error during search: {e}", err=True)
+            return
+        finally:
+            # Close streaming exporters
+            try:
+                if csv_exporter:
```

Ruff (0.12.2)

Undefined name `csv_exporter`

(F821)